### DISCUSSION TITLE
Will not use / import P1S json files

### DISCUSSION THREAD

**Discussion by swdw973** [OrcaSlicer/OrcaSlicer#1640] [General]
Tried to import the P1S files from Bambu Studio. Printer does nte show up.

I get the "0 configs imported (Only non system and compatible configs)" pop up window.

I also tried copying them directly from the BS configs.  
C:\Users\swdw9\AppData\Roaming\BambuStudio\system\BBL\machine
to
C:\Users\swdw9\AppData\Roaming\OrcaSlicer\system\BBL\machine

Still a no go after restarting OrcaSlicer.
What do I have to do to get the P1S to show up?

The reason for wanting to do so is the P1S is a hybrid on settings. When selected in Bambu Studio, it shows as P1S under the selected printer, but all the nozzle listings are for the X1C (w/o the calibration extrusions at the front of the plate).

**Comment by DrDoomsLab** [comment]
If you take a look at some of the json files you can see all Bambu did for the beta was add the P1S to the X1C profile. 

I originally used the P1P profile and it did kick up a fuss but i pressed continue and it seemed to work fine. Print was all good.

I've managed to get the printer and the profiles to show up in orca after copying and editing some of the json files. Am yet to test a print with it as im still waiting for another print to finish from Bambu Studio. Only thing i cant seem to get working in orca like in BS is the image for the P1S on the printer select screen. And no camera. But it wont work for me in BS.

Take a look at the BBL.json file at \resources\profiles\

The calibration extrusions is just an svg image file that shows up on the 3d bed model and they change it depending on the printer used. again set out in one of the json files.

Hope this helps. Cheers.

**Comment by swdw973** [comment]
I think the reason the P1S was added to the X1C, and the reason orca kicked up a fuss when you used P1P, is because the P1S has the closed loop control the X1C has and the P1P doesn't.

Will wait for an update to include the P1S. I'm not as knowledgeable as you are in these areas.
